src/sage/schemes/elliptic_curves/ell_field.py

```diff
@@ -1636,6 +1636,65 @@ def isogenies_prime_degree(self, l=None, max_l=31):
         from .isogeny_small_degree import isogenies_prime_degree
         return sum([isogenies_prime_degree(self, d) for d in L], [])
 
+    def isogenies_degree(self, n):
+        r"""
+        Return a list of all separable isogenies of given degree with domain
+        equal to ``self``, which are defined over the base field of ``self``.
```

Comment:
...up to post-composition with isomorphisms. It may be worth pointing this out (also for `.isogenies_prime_degree()`).